scripts/redirect_url_utils.py

```diff
@@ -83,18 +103,27 @@ def save_map():
 
   print(f'Found redirect files: {total_files}')
 
-if __name__ == '__main__':
-  parser = argparse.ArgumentParser(description="""\
-    Generate bacon.yml file from bacon.json
-    """)
-  parser.add_argument(
-      'method',
-      help='apply_map: load map and create redirects. save_map: create file to url map',
-      nargs='?',
-      default='apply_map')
-  args = parser.parse_args()
-
-  if args.method == 'save_map':
+def run(file=None, save=None):
+  if file:
+    print("Adding new redirects...")
+    update_map(file)
```

Comment:
The next time we call `save_map()`, will it override what was added using `update_map` method?

Reply:
ignore this one